env.d.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 namespace NodeJS {
   interface ProcessEnv {
-    APL: "file" | "vercel" | "upstash";
+    APL: "file" | "vercel" | "upstash" | "REST";
```

Comment:
```suggestion
    APL: "file" | "vercel" | "upstash" | "rest";
```

All names are in the lowercase, lest keep that